lxd/containers.go

```diff
@@ -757,27 +757,20 @@ func (d *lxdContainer) applyConfig(config map[string]string) error {
 	}
 
 	if lxcConfig, ok := config["raw.lxc"]; ok {
-		for _, line := range strings.Split(lxcConfig, "\n") {
-			contents := strings.SplitN(line, "=", 2)
-			key := contents[0]
-			value := ""
-			if len(contents) > 1 {
-				value = contents[1]
-			}
-
-			key = strings.Trim(key, " ")
-			value = strings.Trim(value, " ")
-
-			// empty lines
-			if key == "" {
-				continue
-			}
+		f, err := ioutil.TempFile("", "lxd_config")
```

Comment:
Looks to me like we've been using the trailing underscore everywhere until now:

    stgraber@dakara:~/data/code/lxc/lxd$ grep -r TempFile
    lxc/config.go:	f, err := ioutil.TempFile("", "lxc_profile_")
    lxc/image.go:		f, err := ioutil.TempFile("", "lxc_image_")
    lxd/images.go:	f, err := ioutil.TempFile(dirname, "image_")